consensus/clique/clique.go

```diff
@@ -195,6 +195,11 @@ func New(config *params.CliqueConfig, db ethdb.Database) *Clique {
 	if conf.Epoch == 0 {
 		conf.Epoch = epochLength
 	}
+
+	if conf.ShadowForkHeight > 0 && conf.ShadowForkHeight%conf.Epoch != 0 {
+		panic("ShadowForkHeight must be a multiple of Epoch")
```

Comment:
Maybe throw a Error will better?

Reply:
I would rather not change the function signature. This is just for internal testing anyways.